### DISCUSSION TITLE
Full Power Off as Option?

### DISCUSSION THREAD

**Discussion by rhysbw** [dhleong/playactor#58] [Ideas]
I really like this, but would there be any way to get a full shut down? i want to use it for a scene through homebridge

**Comment by dhleong** [comment]
Unfortunately I believe this has come up before and that's not a feature of the protocol. If you did a full shutdown, there would be no way to wake it again. So, standby is the only option.

**Comment by rhysbw** [comment]
Yes, I understand you couldn’t turn it back on, but is there a way to just
shut it down? Like i could add it as a separate accessory for just a button
that shuts down
On Mon, 2 Jan 2023 at 20:16, Daniel Leong ***@***.***> wrote:

> Unfortunately I believe this has come up before and that's a feature of
> the protocol. If you did a full shutdown, there would be no way to wake it
> again. So, standby is the only option.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/dhleong/playactor/discussions/58#discussioncomment-4575524>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AWMHGSG5JWEKSNJK4BJWXFTWQMZRHANCNFSM6AAAAAATPB7ISI>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>
-- 
Rhys Broughton,
07914757531.

**Reply by dhleong** [reply]
As I said, it's not a feature of the protocol. It's not possible for me to implement something that the console doesn't support. If it's not possible through an official Sony app, it will not be possible through this project.
